plugins/modules/network_settings_workflow_manager.py

```diff
@@ -224,59 +224,86 @@
             type: str
           settings:
             description: Network management details settings.
             type: dict
             suboptions:
               network_aaa:
-                description: Network V2's network_aaa.
+                description: Manages AAA (Authentication Authorization Accounting) for network devices.
                 suboptions:
+                  servers:
+                    description: Server type for managing AAA for network devices.
+                    choices: [AAA, ISE]
+                    default: ISE
+                    type: str
                   ip_address:
-                    description: IP address for AAA and ISE server (eg 1.1.1.1).
+                    description:
+                    - Primary IP address for the ISE server.
+                    - Secondary IP address for the AAA server.
+                    - For example, 1.1.1.11.
                     type: str
                   network:
-                    description: IP Address for AAA or ISE server (eg 2.2.2.2).
+                    description:
+                    - PAN IP address for the ISE server.
+                    - Primary IP address for the AAA server.
+                    - For example, 1.1.1.10.
                     type: str
                   protocol:
-                    description: Protocol for AAA or ISE serve (eg RADIUS).
-                    type: str
-                  servers:
-                    description: Server type for AAA Network (eg AAA).
+                    description: Protocol for AAA or ISE server.
+                    choices: [RADIUS, TACACS]
+                    default: RADIUS
                     type: str
                   shared_secret:
-                    description: Shared secret for ISE Server.
+                    description:
+                    - Shared secret for ISE Server.
+                    - Required when the servers is set to ISE.
+                    - Length of the shared secret should be atleast 4 characters.
                     type: str
                 type: dict
               client_and_endpoint_aaa:
-                description: Network V2's clientAndEndpoint_aaa.
+                description: Manages AAA (Authentication Authorization Accounting) for clients and endpoints.
                 suboptions:
+                  servers:
+                    description:
+                    - Server type for managing AAA for client and endpoints.
+                    choices: [AAA, ISE]
+                    default: ISE
+                    type: str
                   ip_address:
-                    description: IP address for ISE serve (eg 1.1.1.4).
+                    description:
+                    - Primary IP address for the ISE server.
+                    - Secondary IP address for the AAA server.
+                    - For example, 1.1.1.1.
                     type: str
                   network:
```

Comment:
Can we say network_address? 
(OR)
We can make ip_address, pan_address -> for ISE and secondary_ip_address -> for AAA. 
Otherwise it may confuse customers to use ip_address variable